public/app/core/components/PasswordField/PasswordField.tsx

```diff
@@ -25,13 +25,9 @@ export const PasswordField: FC<Props> = React.forwardRef<HTMLInputElement, Props
         aria-label={selectors.pages.Login.password}
         ref={ref}
         suffix={
-          <IconButton
+          <Icon
```

Comment:
can we add `tabIndex={0}` here to allow for tabbing to the show password button?

Reply:
actually a better solution is to keep `IconButton`, but add `type="button"`. that way it won't trigger on enter. see https://stackoverflow.com/a/12914700